2_process.R

```diff
@@ -57,9 +54,7 @@ p2_targets_list <- list(
   # Pair PRMS segments with intersecting NHDPlusV2 reaches and contributing NHDPlusV2 catchments
   tar_target(
     p2_prms_nhdv2_xwalk,
-    create_GFv1_NHDv2_xwalk(prms_lines = p1_reaches_sf,nhd_lines = p1_nhdv2reaches_sf,
-                            prms_hrus = p1_catchments_sf_valid,
-                            min_area_overlap = 0.5,drb_segs_spatial = drb_segs_spatial)
+    read_csv(GFv1_NHDv2_xwalk_url, col_types = 'cccc')
```

Comment:
When I run `tar_make()` I get an error on `p2_natural_baseflow` - that target is relying on a tabular join with key = COMID (where COMID is an integer in the data we grab from SB). Do you get that error too? I'm wondering whether we need to keep `comid_down` as an integer or just specify the COMID column as character within the `p2_natural_baseflow` target (line 27 of 2_process/munge_natural_baseflow.R).

Reply:
I'm in the habit of making IDs characters, primarily because they sometimes start with 0 or are long and get truncated when treated as numeric (not sure if that's true for COMIDs). I'm more in favor of converting the `baseflow_pred_files` to a character COMID. I just searched for "comid_down" and didn't see other code to change.

Reply:
Thanks! I specified the column types for all columns